pytest.ini

```diff
@@ -4,6 +4,8 @@ log_cli_level = INFO
 addopts = --maxfail=1000 -rf
 filterwarnings =
     error
+    ignore::DeprecationWarning:samcli.lib.samlib.wrapper
+    ignore::DeprecationWarning:samtranslator.validator.validator
```

Comment:
sam-t sill deprecating the validator class in the future, so they kept the deprecation warning.